SPECS/ca-certificates/ca-certificates.spec

```diff
@@ -73,6 +73,7 @@ Name:           ca-certificates
 # http://hg.mozilla.org/projects/nss/raw-file/default/lib/ckfw/builtins/certdata.txt
 # (but these files might have not yet been released).
 
+# WHEN UPDATING VERSION/RELEASE: remember to update prebuilt-ca-certificates as well.
```

Comment:
>RELEASE [](start = 24, length = 7)

It's only the version we care about. The releases don't need to be aligned, fortunately.

Reply:
Discussed offline, we do need to update both, my bad.

---
In reply to: [543765725](https://github.com/microsoft/CBL-Mariner/pull/403#discussion_r543765725) [](ancestors = 543765725)

Reply:
I believe the prebuilt package would need a rebulilt too, since new files are arriving.